adapters/adnuntius/adnuntius.go

```diff
@@ -344,6 +350,18 @@ func generateAdResponse(ad Ad, impId string, html string, request *openrtb2.BidR
 		}}
 	}
 
+	price := ad.Bid.Amount
+	priceType, _, _, _ := jsonparser.Get(imp.Ext, "bidder", "priceType")
+
+	if string(priceType) != "" {
+		if strings.ToLower(string(priceType)) == "net" {
```

Comment:
Please prefer to use `strings.EqualFold` for a case insensitive comparison.

Reply:
Thanks, this is changed.